src/openms/source/ANALYSIS/OPENSWATH/SwathWindowLoader.cpp

```diff
@@ -114,21 +117,24 @@ namespace OpenMS
     std::vector<double> & swath_prec_upper_ )
   {
     std::ifstream data(filename.c_str());
-    std::string line;
-
-    // check for presence of header, a header is present if the first line contains alphabetic characters
+    String line;
+    std::vector<String> headerSubstrings;
     double lower, upper;
-    std::getline(data, line); // possible header (or first line)
-    if (std::any_of(line.begin(), line.end(), [](char const &c) { return std::isalpha(c); }))
-    { // if any of the characters are alphabetical then there is a header
-      std::cout << "Read Swath window header: '" << line << "'\n";
-      std::getline(data, line); // first line of swath window
-     }
-    else 
+
+    // Check for presence of header
+    std::getline(data, line);
+    try { // If string can be successfully converted to double (excluding initial spaces) then the first line is not a header
```

Comment:
bracket on new line... check out https://github.com/OpenMS/OpenMS/wiki/Coding-conventions

Reply:
I added this in a commit